atlas/packages/app/src/shared/components/ToggleButton/ToggleButton.styles.tsx

```diff
@@ -0,0 +1,129 @@
+          color: ${colors.white};
+        }
+      `
+    : null
+
+const pressed = ({ pressedDown }: ToggleButtonStyleProps) =>
+  pressedDown
+    ? css`
+        border-color: ${colors.white};
+        color: ${colors.white};
+        background-color: ${colors.blue[500]};
+        &:hover {
+          background-color: ${colors.blue[700]};
+        }
+        &:active {
+          background-color: ${colors.blue[900]};
+        }
+      `
+    : null
+
+const colorsFromProps = ({ variant, pressedDown }: ToggleButtonStyleProps) => {
+  let styles
+  switch (variant) {
+    case 'tertiary': {
+      styles = css`
+        background-color: transparent;
+        border-color: transparent;
+        color: ${colors.blue[500]};
+        &:hover {
+          color: ${colors.blue[300]};
+        }
+        &:active {
+          color: ${colors.blue[700]};
+        }
+      `
+      break
+    }
+    case 'secondary': {
+      styles = css`
+        color: ${colors.white};
+        background-color: ${pressedDown ? colors.blue[500] : colors.black};
+        border-color: ${pressedDown ? colors.white : colors.blue[500]};
+        &:hover {
+          border-color: ${pressedDown ? colors.white : colors.blue[700]};
+          color: ${pressedDown ? colors.white : colors.blue[300]};
+          background-color: ${pressedDown ? colors.blue[700] : ''};
+        }
+        &:active {
+          border-color: ${pressedDown ? colors.white : colors.blue[700]};
+          color: ${pressedDown ? colors.white : colors.blue[700]};
+        }
+      `
+      break
+    }
+    case 'primary':
+    default: {
+      styles = css`
+        color: ${colors.white};
+        background-color: ${colors.blue[500]};
+        border-color: ${pressedDown ? colors.white : colors.blue[500]};
+        &:hover {
+          background-color: ${colors.blue[700]};
+          border-color: ${pressedDown ? colors.white : colors.blue[700]};
+          color: ${colors.white};
+        }
+        &:active {
+          background-color: ${colors.blue[900]};
+          border-color: ${pressedDown ? colors.white : colors.blue[900]};
+          color: ${colors.white};
+        }
+      `
+      break
+    }
+  }
+  return styles
+}
+
+const disabled = ({ disabled }: ToggleButtonStyleProps) =>
+  disabled
+    ? css`
+        box-shadow: none;
+        fill: unset;
+        stroke: unset;
+        color: ${colors.white};
+        background-color: ${colors.gray[100]};
+        border-color: ${colors.gray[100]};
+        &:hover {
+          color: ${colors.white};
+          background-color: ${colors.gray[100]};
+          border-color: ${colors.gray[100]};
+        }
+        &:active {
+          color: ${colors.white};
+          background-color: ${colors.gray[100]};
+          border-color: ${colors.gray[100]};
+        }
+      `
+    : null
+
+export const StyledToggleButton: React.FC<ToggleButtonStyleProps> = (props) => (
```

Comment:
Do we need to define a component explicitly here? I'd think `styled(Button)` would work fine. No need to change it though, just curious

Reply:
I thought so too, but for some reason using `styled(Button)` was not working and no styles were being applied. If you have any idea of why let me know and I'll revert back to that.

Reply:
`Button` needs to accept `className` prop and pass it to its container for styles to be applied